### ISSUE TITLE
Sequelize - Projects using the very recent Sequelize version cannot display the data due to old operators deprecation

### ISSUE THREAD

**Issue by danielres** [ForestAdmin/forest-express-sequelize#161]
## Expected behavior

Collections working in admin

## Actual behavior

Any collection tab in the admin shows error:

> Oops, something went wrong
An error occured while retrieving your data. Please contact support@forestadmin.com for further investigation.

## Failure Logs

This example is for table User, but same problem occurs with all other tables.

```
 db_1       | 2018-01-25 13:04:05.512 UTC [33] STATEMENT:  SELECT count("User"."id") AS "count" FROM "public"."User" AS "User" LEFT OUTER JOIN "public"."Profile" AS "profile" ON "User"."id" = "profile"."userId" WHERE "User"."$and" IN (NULL);
[1] [forest] 🌳🌳🌳  Unexpected error: column User.$and does not exist
[1] SequelizeDatabaseError: column User.$and does not exist
```

## Context

The Forest admin was all working fine before, but stopped working now.

* Package Version: "forest-express-sequelize": "^2.1.1"
* Express Version: "express": "~4.15.5"
* Sequelize Version: "^4.32.1"
* Database Dialect: postgres
* Database Version: postgres (PostgreSQL) 10.1

**Comment by arnaudbesnier** [comment]
Hey @danielres,

Did you upgrade to a new liana version?
It seems that the error appears in a strange segment or filter that generates this weird condition `WHERE "User"."$and" IN (NULL)`.

Can you share a screenshot of the broken UI?
A screenshot of the request/payload in the Chrome console (Network tab) would be also very helpful.

Thank you for the feedback.

**Comment by danielres** [comment]
I made sure liana (`"forest-express-sequelize": "^2.1.1"`) is at the latest version, didn't fix the issue.

Here are the screenshots, same problem with all the collections (Users, Profiles, Projects, Purchases):

![image](https://user-images.githubusercontent.com/168687/35391237-5e6045e6-01de-11e8-9a84-91b24431b13f.png)

![image](https://user-images.githubusercontent.com/168687/35391356-b31b056c-01de-11e8-8c52-659e33bb5757.png)

Thank you for your help.

**Comment by arnaudbesnier** [comment]
Can you share the "Headers" tab of the last screenshot, what is interesting to me is the format of the request and especially the payload sent. 🙏

**Comment by danielres** [comment]
sure 

![image](https://user-images.githubusercontent.com/168687/35392774-3e2dcf82-01e3-11e8-8e5b-4e1e5848b125.png)

**Comment by danielres** [comment]
I've been trying to checkout different commits in the project's git history and find at which point the problem ocurs.

The problem seems to appear with a commit that does this:

Changes in our `init` migration (we keep 1 migration while we're in early dev mode, with dropping & migrating the whole db when the migration is changed, and restarting the server).

```

         CREATE TABLE "User" (
           "id" uuid primary key default uuid_generate_v4(),
+          "authId" varchar NOT NULL UNIQUE,
           "firstName" varchar NOT NULL,
           "lastName" varchar NOT NULL,
```

server/models/User.js file, used by forest and the rest of the app:

```
        primaryKey: true,
       },
+      authId: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
       firstName: {
         type: DataTypes.STRING,
         allowNull: false,
```

Before this change, the forest admin works fine.
Still don't undedrstand why this would be triggering the problem. Am I missing something ?

**Comment by arnaudbesnier** [comment]
Thanks for sharing, it is really weird.
As the latest error your shared was on the "Projects" collection, can you share the SQL select query error on the "projects" table.

**Comment by danielres** [comment]
sure (I replaced some parts with "..." for confidentiality reasons)

```
OPTIONS /forest/Project?fields%5BProject%5D=name%2Cstatus%2Cdescription%2CcreatedAt%2CupdatedAt&page%5Bnumber%5D=1&page%5Bsize%5D=10&searchExtended=0&sort=-createdAt&timezone=Europe%2FBerlin 200 0.202 ms - 2
[1] [forest] 🌳🌳🌳  Unexpected error: column Project.$and does not exist
[1] SequelizeDatabaseError: column Project.$and does not exist
[1]     at Query.formatError (.../node_modules/sequelize/lib/dialects/postgres/query.js:356:16)
[1]     at query.catch.err (.../node_modules/sequelize/lib/dialects/postgres/query.js:86:18)
[1]     at tryCatcher (.../node_modules/bluebird/js/release/util.js:16:23)
[1]     at Promise._settlePromiseFromHandler (.../node_modules/bluebird/js/release/promise.js:512:31)
[1]     at Promise._settlePromise (.../node_modules/bluebird/js/release/promise.js:569:18)
[1]     at Promise._settlePromise0 (.../node_modules/bluebird/js/release/promise.js:614:10)
[1]     at Promise._settlePromises (.../node_modules/bluebird/js/release/promise.js:689:18)
[1]     at Async._drainQueue (.../node_modules/bluebird/js/release/async.js:133:16)
[1]     at Async._drainQueues (.../node_modules/bluebird/js/release/async.js:143:10)
[1]     at Immediate.Async.drainQueues [as _onImmediate] (.../node_modules/bluebird/js/release/async.js:17:14)
[1]     at runCallback (timers.js:773:18)
[1]     at tryOnImmediate (timers.js:734:5)
[1]     at processImmediate [as _immediateCallback] (timers.js:711:5)
[2] db_1       | 2018-01-25 14:32:22.872 UTC [81] ERROR:  column Project.$and does not exist at character 84
[2] db_1       | 2018-01-25 14:32:22.872 UTC [81] STATEMENT:  SELECT count("Project"."id") AS "count" FROM "public"."Project" AS "Project" WHERE "Project"."$and" IN (NULL);
[2] db_1       | 2018-01-25 14:32:22.872 UTC [80] ERROR:  column Project.$and does not exist at character 125
[2] db_1       | 2018-01-25 14:32:22.872 UTC [80] STATEMENT:  SELECT "id", "userId", "status", "name", "description", "createdAt", "updatedAt" FROM "public"."Project" AS "Project" WHERE "Project"."$and" IN (NULL) ORDER BY "Project"."createdAt" DESC LIMIT 10 OFFSET 0;
[1] GET /forest/Project?fields%5BProject%5D=name%2Cstatus%2Cdescription%2CcreatedAt%2CupdatedAt&page%5Bnumber%5D=1&page%5Bsize%5D=10&searchExtended=0&sort=-createdAt&timezone=Europe%2FBerlin 500 12.665 ms - 73
```

**Comment by arnaudbesnier** [comment]
Ok, I think you upgraded your Sequelize version, am I wrong?
I think you forbid old Sequelize operators and that make Forest queries to be malformed.
See: http://docs.sequelizejs.com/manual/tutorial/querying.html#operators-aliases

Do you confirm it could be the reason?

**Comment by danielres** [comment]
indeed, locking sequelize to version `4.8.0` solves the problem (instead of `^4.28.6`).

Thank you for the great support !

**Comment by danielres** [comment]
Seems like sequelize dropping the old operators is a security measure.
It's probably a good idea to update forest accordingly.

**Comment by arnaudbesnier** [comment]
@danielres, you're right, this is something we need to work on.

**Comment by arnaudbesnier** [comment]
Let me reopen this issue, we should definitely support the latest Sequelize versions. It will push us to make it happen ;)

**Comment by arnaudbesnier** [comment]
@danielres @TimAstier The fix is now available in the latest liana release [2.2.1](https://github.com/ForestAdmin/forest-express-sequelize/blob/devel/CHANGELOG.md#release-221---2018-01-29)

Thanks for the feedback,
🌲🌲🌲
